doc/design/profiler.md

```diff
@@ -12,22 +12,22 @@ for i in xrange(M):  # M is  the iteration number
     op.run();
 ```
 
-In summary, the proflier should have following features:
+In summary, the profiler should have the following features:
 
-- records time span in loop.
-- supports nested time span.
-- supports multiple threads/multiple GPUs.
-- supports to be enabled and disabled by users.
+- Record time span in a loop.
```

Comment:
Should be `Records` due to `profiler`. The same below.

Reply:
Thanks! Done.